src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Performance/BasicDoNotUseCountWhenAnyCanBeUsedFixer.vb

```diff
@@ -38,13 +38,13 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Performance
 
                         If Not member Is Nothing Then
 
-                            GetExpressionAndInvocationArguments(
+                            If TryGetExpressionAndInvocationArguments(
                                 sourceExpression:=member.Expression,
                                 isAsync:=isAsync,
                                 expression:=expression,
-                                arguments:=arguments)
-
-                            Return True
+                                arguments:=arguments) Then
+                                Return True
```

Comment:
You can `Return TryGetExpressionAndInvocationArguments` instead, same for the other cases.